Orange/widgets/visualize/owdistributions.py

```diff
@@ -371,8 +371,6 @@ def display_distribution(self):
                 self.tooltip_items.append((self.plot, item))
 
     def _on_relative_freq_changed(self):
-        if not self.distributions:
-            return
```

Comment:
No. If we really wanted to keep this check, we would write `if self.distributions is None or not len(self.distributions)`.

However this check should not be here. `self.distributions` can be empty or `None`, but `self.contingencies` may be filled with values and that is the reason that the code below should be executed.

Crash is now prevented in a different way.